bot/teamsBot.ts

```diff
@@ -160,6 +172,76 @@ export class TeamsBot extends TeamsActivityHandler {
           );
           break;
         }
+        case "mark student complete": {
+          // only one student at a time can be in a conversing state
+          const studentToUpdate: QueueEntry =
+            this.activeQueue.findFirstConversing();
+          if (studentToUpdate != undefined && !this.activeQueue.isEmpty()) {
+            studentToUpdate.setResolvedState(StudentStatus.Resolved);
+            const updateResult = await updateQueueEntryResolved(
+              this.dbConnection,
+              studentToUpdate.id,
+              studentToUpdate.resolved
+            );
+            console.log(`Updated: ${updateResult}`);
+            await context.sendActivity(
+              `Student conversation resolved:${studentToUpdate.toString()}`
+            );
+          } else {
+            await context.sendActivity(
+              "Unable to mark student as completed - there are either no students conversing with an instructor or no students are in line."
+            );
+          }
+          break;
+        }
+        case "get next student": {
+          const anyConversing: QueueEntry =
+            this.activeQueue.findFirstConversing();
+          if (anyConversing != undefined) {
+            await context.sendActivity(
+              `Have you finished helping the other student? Please resolve the conversation with the current student via 'mark student complete'. Currently needs resolving: ${anyConversing.toString()}`
+            );
+          } else if (this.activeQueue.isEmpty()) {
+            await context.sendActivity(
+              "There are currently no students in line!"
+            );
+          } else {
+            const nextInLine = this.activeQueue.findFirstWaiting();
+            if (nextInLine == undefined) {
+              await context.sendActivity(
+                "There are no students in queue looking for help right now."
+              );
+              break;
+            }
+            // mark the next student
+            nextInLine.setResolvedState(StudentStatus.Conversing);
+            const updateResult = await updateQueueEntryResolved(
+              this.dbConnection,
+              nextInLine.id,
+              nextInLine.resolved
+            );
+            console.log(`Updated: ${updateResult}`);
+            await context.sendActivity(
+              `Next student to be helped:${nextInLine.toString()}`
+            );
```

Comment:
Only one change I see. Can you delete this line?

Reply:
How does this look? Explicit response to the command itself followed by tagging the user. 
![image](https://user-images.githubusercontent.com/40576340/156843631-31eab9a3-0faa-4fd1-8c98-4fb5ebb4b6b0.png)